CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 #### Breaking Changes
 
 - `wayland-rs` dependencies are updated to 0.30 and all APIs have changed significantly as a result
+- `xkeysym::Keysym` is used as a keyboard key representation instead of `u32`
```

Comment:
You should put it in the right place.

Reply:
This is a breaking change, I think?

Reply:
I haven't said that it's not, it just for 0.17 which was released months ago. You should move to the actual unreleased section.

Reply:
My bad, fixed!